controller/test/controller/controller_test.cpp

```diff
@@ -0,0 +1,71 @@
+/* Copyright 2020, RespiraWorks
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+#include "controller.h"
+#include "gtest/gtest.h"
+#include <cmath>
+#include <tuple>
+
+// TODO: There ought to be many more tests in here.
+
+TEST(ControllerTest, Volume) {
```

Comment:
Please add a brief comment what this test tests, or rename the test so that it's more clear.

Reply:
Done